CloneTiving/CloneTiving/Extensions/UILabel+.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UILabel.swift
+//  CloneTiving
+//
+//  Created by 지희의 MAC on 2023/05/01.
+//
+
+import UIKit
+
+extension UILabel {
+        func labelWithImg(composition: NSAttributedString...) {
+            let attributedString = NSMutableAttributedString()
+            for i in composition {
+            attributedString.append(i)
+            }
```

Comment:
```suggestion
composition.forEach { attributedString.append($0) }
```
고차함수를 이용해서 바꿔보는것도 좋아보입니다

Reply:
오.. 확실히 깔끔하네요!! 감사합니다